Gopistol/demo/src/main/java/com/example/demo/comment/domain/Comment.java

```diff
@@ -0,0 +1,38 @@
+package com.example.demo.comment.domain;
+
+import com.example.demo.member.domain.Member;
+import com.example.demo.post.domain.Post;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Data
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor
```

Comment:
```suggestion
@AllArgsConstructor(access = AccessLevel.PRIVATE)
```
정적 팩토리 메서드만 외부에 노출시킬 생각이라면 접근제어자 고민해보기